tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
I would merge this if @geerlingguy is fine with it. Not sure if this is the way to go but definitely want this PR to have a happy end. 😄